Assets/Scripts/Editor/ThrowableCheats.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+public class ThrowableCheats : MonoBehaviour
+{
+    [MenuItem(Constants.CHEATS_ADD_MEDICINE_BOTTLES, true)]
+    public static bool ValidateAddMedicineBottles()
+    {
+        return Application.isPlaying && UIManager.Instance;
+    }
+
+    [MenuItem(Constants.CHEATS_ADD_MEDICINE_BOTTLES)]
+    public static void AddMedicineBottles()
+    {
+        GameObject medicineBottlePrefab = Resources.Load<GameObject>("Cheats/Medicine_Bottle_Throwable_CHEATS");
```

Comment:
Why do we need a specific game object for cheats? If these diverging, might be a pain to upkeep both.

Reply:
The tricky part was that cheats need to access everything from a static context, so I can't drag the regular bottle prefab into the inspector.  The only option I could find was the Resources.Load() method, and that required that the prefab be located in a folder called "Resources".  Instead of moving Medicine_Bottle_Throwable, I made a prefab variant which automatically updates from the original (so no extra upkeep required)

Reply:
I see! Well if it's a prefab variant, then that'll work!! Thanks for clearing that up :)